app/views/govuk_component/option_select.raw.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="govuk-option-select" tabindex="0">
+  <div class="container-head js-container-head">
+    <div class='option-select-label'><%= title %></div>
+  </div>
+  <div class='options-container'>
+    <div class='js-auto-height-inner'>
+      <% options.each do | option |%>
+
+      <label for='<%= option['id'] %>'>
```

Comment:
I am slightly concerned about the potential for an attack on this `label` and the `input` below. In theory if an application allowed the options to be user generated I think user could enter a string like `' onclick="nasty"'` which wouldn't get encoded as part of the standard rails string escaping. Using something like `content_tag(:label, for: option["id"])` would add the extra escaping to protect the page.

I could well be wrong here though and am happy to be shown that the standard escaping works.